_posts/guides/2022-04-29-guide_porting-goldsrc-to-source.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Porting Goldsrc maps to Source
+
+permalink: /guide/porting-goldsrc-maps-to-source/
+
+category: guide
+
+tags:
+  - bhop
+  - climb
+  - conc
+  - mapping
+  - porting
+  - guide  
+
+toc: true
+
+toc_sticky: true
+---
+Before you start, you should know that maps with simple geometry work best for porting. Complex geometry can cause invalid solids which will need to be manually remade, and some textures will need to be realigned.
+
+**Prerequisites**:
+- Have a basic understanding of Goldsrc, Valve Hammer Editor, and potentially Trenchbroom
+ 
+## Download the .BSP and (in some cases) the .WAD
+
+You can download maps from [Xtreme-Jumps](https://xtreme-jumps.eu/download.php?list.9) or by joining an Adrenaline Gamer Kreedz server, and switcihg the map with `agmap.` The BSP is the actual map file, and the WAD contains the textures used in the map, although in most cases custom textures will be embedded into the BSP.
+
+## Obtaining the .map file
+ 
+You will need the .map/.rmf/.jmf file for the map you wish to port to Source. It will make your life much easier if you can get the original from the creator, however it is not required.
+
+## Decompiling with WinBSPC
+
+Open [WinBSPC](https://valvedev.info/tools/winbspc/), in the top left corner select file > convert. Navigate to the BSP file you wish to decompile and select it. Choose Convert to: MAP, and best match texturing. Choose the desired output folder and click OK.
+
+![DECOMPILE PARAMETERS](/assets/images/goldsrc_to_source_guide/WINBSPC_PARAMETERS.png)
+
+## Decompiling with BSPTwoMap
+
+[BSPTwoMap](https://valvedev.info/tools/bsptwomap/), is a command-line application, so simply drag the bsp onto to .exe to decompile it.
+
+## What decompiler to use?
```

Comment:
Is this also a subsection of `Obtaining the .map file`? Also, this section should probably come before the two tools.